watchtower/blob/justice_kit_test.go

```diff
@@ -27,10 +27,20 @@ func makeSig(i int) lnwire.Sig {
 	return sig
 }
 
+func makeAddr(size int) []byte {
+	addr := make([]byte, size)
+	if _, err := io.ReadFull(rand.Reader, addr); err != nil {
```

Comment:
We may want to consider using the `math/rand` reader here. Not a blocker atm, but if we switch to this source, then the unit tests become I/O free as they no longer have to obtain any file descriptors etc. For the sake of moving forward, I think a TODO here is enough, possibly a blanket issue within `lnd` to do a sweep and remove instances of `crypto/rand` that aren't actually required.